hw/ip/adc_ctrl/data/adc_ctrl.prj.hjson

```diff
@@ -11,6 +11,6 @@
     version:            "1.0",
     life_stage:         "L1",
     design_stage:       "D2S",
-    verification_stage: "V1",
+    verification_stage: "V2S",
```

Comment:
@weicaiyang are you declaring V2 or V2S for this one?

Reply:
Yes, we can move directly to V2S